### PR TITLE
chore(2022-12-14-term-test): add init values

### PR DESCRIPTION
**Pull request by jrhender**
Add intialization values for termination-test contract

### CODE REVIEW

**Review by knzeng-e — APPROVED** [review]

File: scripts/pool-deployments/2022-12-14-ewc-termination-test/2022-12-14-ewc-termination-test.env [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
\ No newline at end of file
+CLAIM_MANAGER_ADDRESS=0x23b026631a6f265d17cfee8aa6ced1b244f3920c
+
+# #########
+# 2. Initialization parameters
+# For this contract, initialization be performed manually
+# RATIO = 0.004
+# HARD_CAP = 10
+# END_DATE = 1671125400 Date and time (GMT): Thursday, December 15, 2022 5:30:00 PM
+# START_DATE = 1671039000 (Date and time (GMT): Wednesday, December 14, 2022 5:30:00 PM)
```
Comment (knzeng-e) [inline scripts/pool-deployments/2022-12-14-ewc-termination-test/2022-12-14-ewc-termination-test.env:14]: I finally used those dates -->

 END_DATE = 1671125460
START_DATE = 1671039000

File: scripts/pool-deployments/2022-12-14-ewc-ui-test/2022-12-14-ewc-ui-test.env [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+INITIATOR_ADDRESS=0x7fb7225fffa4bf7bead7d2acfe0d401546d9d809
+CLAIM_MANAGER_ADDRESS=0x23b026631a6f265d17cfee8aa6ced1b244f3920c
+
+# #########
+# 2. Initialization parameters
+# For this contract, initialization be performed manually
+# RATIO = 0.004
+# HARD_CAP = 30
+# END_DATE = 1671125400 Date and time (GMT): Thursday, December 15, 2022 5:30:00 PM
```
Comment (knzeng-e) [inline scripts/pool-deployments/2022-12-14-ewc-ui-test/2022-12-14-ewc-ui-test.env:13]:
```suggestion
# END_DATE = 1671125460 Date and time (GMT): Thursday, December 15, 2022 5:31:00 PM
```

**Review by knzeng-e — APPROVED** [review]
Updating the final dates, and we should be fine. Thank you @jrhender !

File: scripts/pool-deployments/2022-12-14-ewc-ui-test/2022-12-14-ewc-ui-test.env (hunk 2 above)
Comment (jrhender) [reply scripts/pool-deployments/2022-12-14-ewc-ui-test/2022-12-14-ewc-ui-test.env]: Updated 👍

**Pull request merged**